app/views/NirmsNumberView.scala.html

```diff
@@ -14,36 +14,44 @@
  * limitations under the License.
  *@
 
+@import viewmodels.LabelSize._
 @import viewmodels.InputWidth._
+@import models.Mode
 
 @this(
     layout: templates.Layout,
     formHelper: FormWithCSRF,
-    govukErrorSummary: GovukErrorSummary,
     govukInput: GovukInput,
-    govukButton: GovukButton
+    govukButton: GovukButton,
+    govukErrorSummary: GovukErrorSummary
 )
 
+
 @(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(form, messages("nirmsNumber.title"))) {
+@layout(
+    pageTitle    = titleNoForm(messages("nirmsNumber.title")),
+    showBackLink = true
+) {
+
 
-    @formHelper(action = routes.NirmsNumberController.onSubmit(mode)) {
+    @formHelper(action = routes.NirmsNumberController.onSubmit(mode), Symbol("autoComplete") -> "off") {
```

Comment:
What does setting autoComplete to off do? Could it be useful for accessibilty to have it turned on?